openedx/core/djangoapps/course_groups/team_partition_scheme.py

```diff
@@ -0,0 +1,149 @@
+"""
+Provides a UserPartition driver for teams.
+"""
```

Comment:
Similarly, this partition scheme is a plugin to the `course_groups` partitioning interface. Because this scheme is specific to teams and uses LMS models, I recommend moving it to lms/djangoapps/teams/partition_scheme.py.